homeassistant/components/weather/__init__.py

```diff
@@ -19,6 +19,7 @@
 ATTR_FORECAST = "forecast"
 ATTR_FORECAST_CONDITION = "condition"
 ATTR_FORECAST_PRECIPITATION = "precipitation"
+ATTR_FORECAST_PRECIPITATION_PROB = "precipitation_prob"
```

Comment:
Changing our entity model requires approval in an issue in our architecture repo.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Added in [proposal](https://github.com/home-assistant/architecture/issues/385)